src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -657,29 +657,32 @@ export const processCreateFormInWorkspace = async (
   return form
 }
 
+type DuplicateFormOpts = {
+  // The ID of the workspace to duplicate the form into.
+  workspaceId?: string
+  // If true, overrides the emails of MRF workflows and dropdown fields with the given overrideEmails
```

Comment:
nit: should the comment be, If `defined` instead of true?

Reply:
Great point! good catch :)

```suggestion
  // If defined, overrides the emails of MRF workflows and dropdown fields with the given overrideEmails
```